app/views/message_threads/bulk_actions.turbo_stream.erb

```diff
@@ -1,3 +1,3 @@
 <%= turbo_stream.replace "bulk_actions" do %>
-  <%= render MessageThreadsBulkActionsComponent.new(ids: @ids) %>
+  <%= render MessageThreadsBulkActionsComponent.new(ids: @ids, user_is_signer: @user_is_signer) %>
```

Comment:
Keby sme ten `Current.user.signer?` dali inline tu, tak by sa nieco zle stalo?

Reply:
dam inline. nic zle sa asi nestane. zvycajne sa ale snazim co najviac vyhybat nejakym veciam, ktore su natvrdo vo view. ale tu to asi prezijeme.